scripts/common/rook.sh

```diff
@@ -100,9 +100,16 @@ function remove_rook_ceph() {
     printf "Data within /var/lib/rook, /opt/replicated/rook and any bound disks has not been freed.\n"
 }
 
-# scale down prometheus, move all 'rook-ceph' PVCs to 'longhorn', scale up prometheus
-function rook_ceph_to_longhorn() {
-    report_addon_start "rook-ceph-to-longhorn" "v1"
+# scale down prometheus, move all 'rook-ceph' PVCs to provided storage class, scale up prometheus
+# Supported storage class migrations from ceph are: 'longhorn' and 'openebs'
+function rook_ceph_to_sc_migration() {
+    local destStorageClass=$1
+    report_addon_start "rook-ceph-to-${destStorageClass}-migration" "v2"
+
+    # we only support migrating to 'longhorn' and 'openebs' storage classes
+    if [ "$destStorageClass" != "longhorn" ] && [[ "$destStorageClass" != *"openebs"* ]]; then
```

Comment:
how do you know that destStorageClass will always be named with openebs? isnt this [configurable](https://github.com/replicatedhq/kurlkinds/blob/5a5061015024d0ecc03b776d74901a6e7b0709ab/config/crds/v1beta1/cluster.kurl.sh_installers.yaml#L1948) by the vendor? you should get the provisioner and use that.

Reply:
Yup. The storage class can be overwritten by the user. I'll change this.